src/include/mpi.h.in

```diff
@@ -879,10 +879,11 @@ typedef int (MPIX_Grequest_wait_function)(int, void **, double, MPI_Status *);
 #define MPI_T_ERR_INVALID_NAME      73  /* Name doesn't match */
 #define MPI_T_ERR_INVALID           74  /* Generic error code for MPI_T added in MPI-3.1 */
 
+#define MPI_ERR_EAGAIN              75  /* Operation could not be issued */
```

Comment:
Should this go below as `MPICH_ERR_EAGAIN` since it's not a constant in the MPI Standard?